src/nora/components/NoraTextAreaInput/NoraTextAreaInput.js

```diff
@@ -17,7 +18,76 @@ export const NoraTextAreaInput = ({
   onChange,
   ...rest
 }) => {
+  const {
+    formChangeHandler,
+    validator,
+    initialValue,
+    currentValue,
+    currentError,
+    setFieldTouched,
+  } = { ...rest }
+
   const classes = [styles.NoraTextInput, textClassName].join(' ')
+
+  // Set up validation hooks
+  const [getError, setError, , validate] = useErrorMessage(validator)
+
+  const [val, setVal] = useState(currentValue || initialValue || '')
+
+  const [touched, setTouched] = useState(initialValue ? true : false)
```

Comment:
What's the point of the logic here?

`NoraTextAreaInput` has a `value` prop which should be the default value. However, in this case, even when there is a default `value,` `touched` is `false` because `initialValue` is falsey.

Shouldn't we be checking `value` to double check whether this field is actually `touched`?